### PR TITLE
Do not launch browser when running integration tests

### PR DESCRIPTION
**Pull request by Zillode**

### CODE REVIEW

**Comment by Zillode** [comment]
@st-jenkins run integration tests for windows

File: test/syncthingprocess.go [hunk 1]
```diff
@@ -66,7 +66,8 @@ func (p *syncthingProcess) start() error {
 		binary = binary + "-" + p.instance + ".exe"
 	}
 
-	cmd := exec.Command(binary, p.argv...)
+	argv := append(p.argv, "-no-browser")
+	cmd := exec.Command(binary, argv...)
```
Comment (calmh) [inline test/syncthingprocess.go]: Cleaner to add this to `env` higher up

**Comment by calmh** [comment]
:+1: Good change, this has annoyed me too :)

File: test/syncthingprocess.go (hunk 1 above)
Comment (Zillode) [inline test/syncthingprocess.go]: confused, this is not an `env` setting but a cmdline argument:o

Comment (calmh) [inline test/syncthingprocess.go]: HERP DERP :stuck_out_tongue_winking_eye: yeah you're right

**Comment by AudriusButkevicius** [comment]
:tophat: this was so annoying.

**Pull request merged**